src/nixpkgs.rs

```diff
@@ -24,6 +24,7 @@ use crate::package::{PathOrigin, StorePath};
 /// The function returns an Iterator over the packages returned by nix-env.
```

Comment:
You may want to update the help doc above this to include info on the `system` parameter